src/Core/Migration/V6_7/Migration1742897274RegistrationSalutationToggleConfig.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_7;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+use Shopware\Core\Framework\Uuid\Uuid;
+
+/**
+ * @internal
+ */
+#[Package('core')]
+class Migration1742897274RegistrationSalutationToggleConfig extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1742897274;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $connection->insert('system_config', [
+            'id' => Uuid::randomBytes(),
+            'configuration_key' => 'core.loginRegistration.showSalutation',
+            'configuration_value' => json_encode(['_value' => true]),
+            'sales_channel_id' => null,
+            'created_at' => (new \DateTime())->format(Defaults::STORAGE_DATE_TIME_FORMAT),
+        ]);
```

Comment:
This will overwrite existing configuration, if applied again. Please do a quick check, if `core.loginRegistration.showSalutation` with `salesChannelId IS NULL` already exists and then early return.